docs/Platforms.md

```diff
@@ -18,6 +18,8 @@ Optionally, `msquic.dll` can be built with OpenSSL (see below for more details)
 
 On Linux, MsQuic relies on [OpenSSL](https://www.openssl.org/) for TLS 1.3 functionality.
```

Comment:
```suggestion
On Linux, MsQuic relies on QuicTLS, a fork of [OpenSSL](https://www.openssl.org/) for TLS 1.3 functionality. QuicTLS is a small delta on top of OpenSSL that adds APIs to enable the QUIC handshake that establishes TLS in first packet as part of connection establishment.
```

Reply:
We mention QuicTLS a couple lines below.

Reply:
I think the initial statement is partly what caused confusion as to whether the openSSL installed as part of the distro is consumed by MSQuic.

Reply:
Actually, I'm wondering if we could create additional alternative packages where OpenSSL 1.1 _is_ linked to MsQuic. So users could choose depending on their needs and base distro. For now, users requiring FIPS would just need to stay on older version but since the certification process is very long it may be required anyway.